### PR TITLE
Breakdown key simplification cleanups

### PR DESCRIPTION
**Pull request by martinthomson**
@benjaminsavage, this might help.  I've added a couple of tweaks (see #724) above yours as well, because I'll be needing those.

### CODE REVIEW

File: src/protocol/ipa/mod.rs [hunk 1]
```diff
@@ -322,80 +319,6 @@ where
     }
 }
 
-pub struct IPAModulusConvertedInputRow<F: Field, T: LinearSecretSharing<F>> {
```
Comment (martinthomson) [inline src/protocol/ipa/mod.rs:325]: Dead code removal!

File: src/protocol/ipa/mod.rs [hunk 2]
```diff
@@ -492,30 +415,23 @@ where
 
     let arithmetically_shared_values = m_ctx.upgrade(arithmetically_shared_values).await?;
 
-    let binary_shared_values = upgraded_gf2_match_key_bits
-        .iter()
-        .zip(upgraded_gf2_breakdown_key_bits.iter())
-        .map(|(match_key, breakdown_key)| {
-            BinarySharedIPAInputs::new(match_key.clone(), breakdown_key.clone())
-        })
+    let binary_shared_values = zip(upgraded_gf2_match_key_bits, upgraded_gf2_breakdown_key_bits)
```
Comment (martinthomson) [inline src/protocol/ipa/mod.rs:418]: Simplifications here.

**Review by martinthomson — COMMENTED** [review]
Most of this is import cleanups, but I've noted the increments over #725

**Review by akoshelev — APPROVED** [review]

File: src/protocol/ipa/mod.rs (hunk 1 above)
Comment (benjaminsavage) [reply src/protocol/ipa/mod.rs:325]: Awesome :)

**Pull request merged**